### ISSUE TITLE
Add numeric content type

### ISSUE THREAD

**Issue by manuxi** [sulu/sulu#5287]
# Problem description

I'd like to give the content manager the posibility to add e.g. a price which I'd like to format in the template. It could also be possible to use numerics for other tasks (e.g. range slider). At the moment only a text_line can be used for those tasks. If there is a posibility to enter stuff wrong, someone will do.

# Proposed solution

Add an integer content type
Add an float content type

**Comment by danrot** [comment]
What is wrong with the already existing `number` content type?

```xml
<property name="price" type="number" />
```

**Comment by plozmun** [comment]
Maybe it's missing in [Sulu Docs](https://docs.sulu.io/en/2.0/reference/content-types/index.html)?

**Comment by manuxi** [comment]
Oh, I haven't seen it in the docs.

**Comment by danrot** [comment]
Yeah, seems to be missing in the docs... Going to make this a docs issue then.
